routes/app/app-manager.js

```diff
@@ -66,6 +66,22 @@ exports.getAppByID = function(req, res, next) {
         });
 };
 
+/**
+ * Enable an app if it is disabled and vice versa.
+ */
+exports.toggleAppByID = function(req, res, next) {
```

Comment:
If the `modifyApp` function detects the presence of the url parameter `ap_state` then it will attempt to call the `setState` function